
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

"""
@author: mercicle
"""

##########################################
###### Install Python Libraries ##########
##########################################

# !pip install dwave-ocean-sdk networkx
# !pip install numpy --upgrade
# !pip install biopandas

# This is important pymol read_mol2_from_list will error without it.
# !pip install pandas==0.24
# !pip install git+git://github.com/rasbt/biopandas.git
# !pip install pandas-compat

# for .tar files
# !pip install sh glob

# Python Tools for Computational Molecular Biology 
# https://biopython.org/
# !pip install biopython

# Annotating draft bacterial and viral genomes. It may also be used for quickly detecting 
# arbitrary sequences such as antibiotic resistance genes (AMR) proteins in nucleotide 
# sequences. It uses Blast to find hits to known gene sequences from sequence databases. 
# The inputs are fasta files. Annotation is performed in a similar manner to Prokka and 
# first requires an assembled genome if you have sequenced reads.
# https://github.com/dmnfarrell/pathogenie

# !pip3 install -e git+https://github.com/dmnfarrell/pathogenie.git#egg=pathogenie
# Error 

# https://pypi.org/project/pybioviz/
# Bioinformatics visualization tools with PyViz Panel and Bokeh. 
# The objective is to have a set of plots and dashboards that can 
# be re-used inside Jupyter notebooks as part of bioinformatic 
# workflows or deployed as local web apps.
# !pip install pybioviz

# A Python framework for the analysis and visualization of trees.
# and Phylogenomic (intersection of the fields of evolution and genomics) Tools
# http://etetoolkit.org/

# !pip install ete3
# Error 

##########################################
########### DWave Ocean API  #############
##########################################

# Step 1: From https://docs.ocean.dwavesys.com/en/stable/overview/sapi.html#sapi-access, you have to run from terminal
# $ dwave config create
# and provide your API token and other info
    
    
    